src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,62 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String color;
+    private String model;
+    private int speed;
+    private boolean isRunning;
+    private Engine engine;
 
+    Car(String color, String model, int speed, boolean isRunning, core.basesyntax.Engine engine) {
```

Comment:
```suggestion
    public Car(String color, String model, int speed, boolean isRunning, Engine engine) {
```

Reply:
Sorry didn't notice, I don't know why it made it with core.basesyntax, but now I have changed everything!